docker-compose.yml

```diff
@@ -1,8 +1,11 @@
-# Docker compose configuration file for the HMDA Platform
-# Version: 1.0.0
-
-hmda:
-  build: .
-  ports:
-    - 8080:8080
-    - 2551:2551
+# Assumes UI repo is cloned in parent directory
+version: '2'
+services:
+  api:
+    build: .
+  ui:
+    build: ../hmda-platform-ui
+    ports:
+      - "80:80"
+    depends_on:
```

Comment:
When I run `docker exec -ti hmdaplatform_ui_1 /bin/bash`, I can `curl http://api:8080/` and see this:
```
root@c540789ba433:/usr/src/app# curl http://api:8080/
{
  "status": "OK",
  "service": "hmda-api",
  "time": "2016-07-29T15:33:24.876Z",
  "host": "44e2e13fb69c"
}root@c540789ba433:/usr/src/app#
```

Also, I can see the correct api responses in the browser by visiting `/api`, `/api/institutions`, etc.